client/tests/integration/components/packages/applicant-team-editor-test.js

```diff
@@ -0,0 +1,41 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import EmberObject from '@ember/object';
+
+module('Integration | Component | packages/applicant-team-editor', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function(assert) {
```

Comment:
I'm going to look at some ways of thinking about components that may/may not depend on models.

One thing I'm afraid of is having a component require instances of a certain class. Will think on it some more...